src/config.py

```diff
@@ -191,6 +191,8 @@ def load_data(self):
             "name": self._select_val("general", "name"),
             "max_game_length_s": float(self._select_val("general", "max_game_length_s")),
             "max_consecutive_fails": int(self._select_val("general", "max_consecutive_fails")),
+            "max_runtime_before_break_m": False if not self._select_val("general", "max_runtime_before_break_m") else float(self._select_val("general", "max_runtime_before_break_m")),
```

Comment:
I haven't gotten around to refactoring the others, but for new config vars lets try to start using `_default_iff`. see params like "min_gold_to_pick" etc